gremlin-groovy-test/src/main/groovy/org/apache/tinkerpop/gremlin/process/traversal/step/filter/GroovyRangeTest.groovy

```diff
@@ -108,5 +108,15 @@ public abstract class GroovyRangeTest {
         public Traversal<Vertex, Map<String, String>> get_g_V_asXaX_out_asXbX_out_asXcX_selectXa_b_cX_byXnameX_rangeXlocal_1_2X() {
             new ScriptTraversal<>(g, "gremlin-groovy", "g.V.as('a').out.as('b').out.as('c').select('a','b','c').by('name').range(local,1,2)")
         }
+
+        @Override
+        public Traversal<Vertex, Long> get_g_V_skipX2X_count() {
+            new ScriptTraversal<>(g, "gremlin-groovy", "g.V.skip(2).count()")
```

Comment:
I would replace this test with one that is an `order().by(...).skip(2)`-style test so you know its actually skipping the "first 2" and not just giving you two results back. Likewise for your `skip(local)`-test.